grub2/install

```diff
@@ -147,6 +147,14 @@ if [ "$SYS__BOOTLOADER__TRUSTED_BOOT" = yes -a -d "/usr/lib/trustedgrub2/$target
   trusted="--directory=/usr/lib/trustedgrub2/$target"
 fi
 
+if [ "$target" == "powerpc-ieee1275" ] ; then
```

Comment:
Please use `=` for consistency here.

Reply:
yes, it should be =, some versions of test complain about ==